lego/apps/events/serializers/pools.py

```diff
@@ -55,6 +56,10 @@ class PoolAdministrateSerializer(PoolReadAuthSerializer):
     registrations = RegistrationReadDetailedSerializer(many=True)
 
 
+class PoolAdministrateExportSerializer(PoolReadAuthSerializer):
```

Comment:
This is supposed to be a `noop` as well?

And isn't it better to extend `PoolAdministrateSerializer` instead?

Reply:
Ahh, nvm. different user serializer